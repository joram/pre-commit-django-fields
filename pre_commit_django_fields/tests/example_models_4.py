from django.db.models import Model, CharField, TextField, DateTimeField


class ExampleModel(Model):
    name = CharField(max_length=255)
    description = TextField()
    created_at = DateTimeField(auto_now_add=True)
    updated_at = DateTimeField(auto_now=True)
