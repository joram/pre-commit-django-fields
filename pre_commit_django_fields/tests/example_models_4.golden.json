[
  {
    "class_name": "ExampleModel",
    "field_type": "CharField",
    "filename": "example_models_4.py",
    "expected_field_classes": null,
    "lineno": 5,
    "name": "name"
  },
  {
    "class_name": "ExampleModel",
    "field_type": "TextField",
    "filename": "example_models_4.py",
    "expected_field_classes": null,
    "lineno": 6,
    "name": "description"
  },
  {
    "class_name": "ExampleModel",
    "field_type": "DateTimeField",
    "filename": "example_models_4.py",
    "expected_field_classes": null,
    "lineno": 7,
    "name": "created_at"
  },
  {
    "class_name": "ExampleModel",
    "field_type": "DateTimeField",
    "filename": "example_models_4.py",
    "expected_field_classes": null,
    "lineno": 8,
    "name": "updated_at"
  }
]